SIL.Core.Desktop/SIL.Core.Desktop.csproj

```diff
@@ -20,7 +20,7 @@
   <ItemGroup>
     <ProjectReference Include="..\SIL.Core\SIL.Core.csproj" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <Reference Include="System.Configuration" />
     <Reference Include="System.Management" />
     <Reference Include="System.Security" />
```

Comment:
@josephmyers I assume that this conditional means that `netstandard2.0` includes these references by default and they don't needed to be explicitly included?

Reply:
That's correct